frontend/src/routes/+page.svelte

```diff
@@ -5,7 +5,7 @@
 </script>
 
 <div>
-	<WhiteBanner title="Looney" description="Please Login or Register" displayMenuButton=0/>
+	<WhiteBanner title="Looney" description="Please Login or Register" displayMenuButton=0 imgExtraPath=""/>
```

Comment:
Is it necessary to assign empty value to the `imgExtraPath`? If yes, can you assign it as a default value in the component itself instead of putting it over here?